skglm/solvers/fista.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+from scipy.sparse import issparse
+from numba import njit
+from skglm.solvers.base import BaseSolver
+from skglm.solvers.common import construct_grad, construct_grad_sparse
+
+
+@njit
```

Comment:
cosmetics, can you put this function below the class?

Reply:
or in skglm.utils ?